tests/test_mathematical.py

```diff
@@ -194,17 +194,11 @@ def test_floor_divide(self, dtype, lhs, rhs):
     def test_hypot(self, dtype, lhs, rhs):
         self._test_mathematical("hypot", dtype, lhs, rhs)
 
-    @pytest.mark.usefixtures("allow_fall_back_on_numpy")
-    @pytest.mark.parametrize(
-        "dtype", get_all_dtypes(no_bool=True, no_complex=True)
-    )
+    @pytest.mark.parametrize("dtype", get_all_dtypes())
     def test_maximum(self, dtype, lhs, rhs):
         self._test_mathematical("maximum", dtype, lhs, rhs, check_type=False)
 
-    @pytest.mark.usefixtures("allow_fall_back_on_numpy")
-    @pytest.mark.parametrize(
-        "dtype", get_all_dtypes(no_bool=True, no_complex=True)
-    )
+    @pytest.mark.parametrize("dtype", get_all_dtypes())
     def test_minimum(self, dtype, lhs, rhs):
```

Comment:
Could you please add tests for `dpnp.fmax` and `dpnp.fmix` ?
These tests used to cover implementation of `dpnp_maximum` and `dpnp_minimum`  cause they were called via `dpnp.maximum/minimum`.
Now `dpnp_maximum` and  `dpnp_minimum` is called via `dpnp.fmax/fmin` and the test coverage is now worse